mteb/abstasks/AbsTaskClassification.py

```diff
@@ -144,16 +145,13 @@ def _evaluate_monolingual(
             scores_exp, test_cache = evaluator(model, test_cache=test_cache)
             scores.append(scores_exp)
 
-        if self.n_experiments == 1:
-            return scores[0]
-        else:
-            avg_scores = {k: np.mean([s[k] for s in scores]) for k in scores[0].keys()}
-            std_errors = {
-                k + "_stderr": np.std([s[k] for s in scores]) for k in scores[0].keys()
-            }
-            return {**avg_scores, **std_errors}
+        avg_scores: dict[str, Any] = {
+            k: np.mean([s[k] for s in scores]) for k in scores[0].keys()
+        }
+        avg_scores["scores_per_experiment"] = scores
```

Comment:
I could imagine we would want to standardize this across task types. If so do we want it to be a list of dicts of a dict with lists

Reply:
I am more for the first option (otherwise the we would have e.g. accuracy and accuracies)

Reply:
The current format:
```
{
task_name: ...,
eval_time: ..,
lang: { metric: score, ...}
...
}
```
Is okay if we expand it to monolingual tasks also. Unless you have a better suggestion

Reply:
@imenelydiaker this section is more on the format of the scores dict on whether is should be:

```
{"accuracy": 0.5, "accuracies": [0.4, 0.6]}
# or
{"accuracy": 0.5, "scores_per_experiment": [{accuracy": 0.4} , {accuracy": 0.6}]}
``` 
I prefer the second

Reply:
However, I would love your opinion on the result format below

Reply:
Agreed the second one with the list is better

Reply:
Agree

Reply:
+1 on 2nd

Reply:
Is this only done for the main metric? If so, it's more space efficient to use something like
```
{"accuracy": 0.5, "scores_per_experiment": [0.4, 0.6]}
```

Reply:
It is done for all scores